dynamic_forms/templates/dynamic_forms/widgets/subfield.html

```diff
@@ -1,4 +1,6 @@
+{% if label != '' %}
```

Comment:
Shouldnt this be `{% if label %}`, so that `None` isn't accidentally printed? I don't recall how Django templating handles this off the top of my head.

Reply:
Right now should don't print None with those fixes I replaced it with a empty string